### DISCUSSION TITLE
Full heart rate history

### DISCUSSION THREAD

**Discussion by paulbrock** [jlai/fitness-dashboard#82] [Ideas]
Is it possible to show the full heart rate history?  I can see resting heart rate history and heart rate zones, but would like just the raw HR history for say the last 7 days.  Sometimes its good to pick out trends or any changes that aren't picked up as individual activities.

**Comment by jlai** [comment]
The API for getting heart rate only allows requesting 24 hours at a time. We could make separate requests to get the last 7 days, although I generally try to avoid making a lot of requests on the main dashboard because it could exhaust the (per-user) hourly rate limit.

Is there a specific place you'd like to see the raw heart rate history?

**Reply by paulbrock** [reply]
Thanks, I was looking for it on the drop down under History Graphs, I think that's the most obvious.
